### ISSUE TITLE
Bad headers for assets

### ISSUE THREAD

**Issue by Broutard** [DarkaOnLine/L5-Swagger#67]
`Resource interpreted as Stylesheet but transferred with MIME type text/html: "/docs/asset/swagger-ui.css".`

So with laravel-debugbar, the debugbar script is included in this css file and break the stylesheet.

**Comment by Broutard** [comment]
You should do that :
```
<?php

namespace L5Swagger\Http\Controllers;

use Illuminate\Routing\Controller as BaseController;
use Illuminate\Http\Response;

class SwaggerAssetController extends BaseController
{
    public function index($asset)
    {
        $path = swagger_ui_dist_path($asset);

        $response = new Response(
            file_get_contents($path), 200, [
                'Content-Type' => 'text/css',
            ]
        );

        return $this->cacheResponse($response);
    }

    /**
     * Cache the response 1 year (31536000 sec)
     */
    protected function cacheResponse(Response $response)
    {
        $response->setSharedMaxAge(31536000);
        $response->setMaxAge(31536000);
        $response->setExpires(new \DateTime('+1 year'));

        return $response;
    }
}

```

**Comment by DarkaOnLine** [comment]
I am aware of that. But this function also serves JS files. Would you like to make a PR for this?

**Comment by DarkaOnLine** [comment]
Fixed in 1df5c11f7d500039038b7b7d5bbddafc657b0929
